src/app/modules/navbar/navbar.component.ts

```diff
@@ -0,0 +1,64 @@
+import { Component, OnInit, ViewChild } from '@angular/core';
+import { Router, NavigationEnd } from '@angular/router';
+import { UserService } from 'src/app/shared/services/user.service';
+import {filter, take} from 'rxjs/operators';
+
+@Component({
+  selector: 'app-navbar',
+  templateUrl: './navbar.component.html',
+  styleUrls: ['./navbar.component.scss']
+})
+export class NavbarComponent implements OnInit {
+
+  @ViewChild('menuModal') menuModal;
+  showNav = true;
+  currentUrl: any;
+  currentUser: any;
+
+  constructor(private router: Router, private userService: UserService) { }
+
+  ngOnInit() {
+  this.router.events.subscribe(event => {
+    if (event instanceof NavigationEnd) {
+      this.currentUrl = this.findUrl(event.url);
```

Comment:
@Nishanthh What is this for?

Reply:
Inorder to find the current route and as per the route i need to show the navbar(variations).

Reply:
@Nishanthh Do this only if we want the user to show the landing page even if he is logged in, if that's not the case, I believe you can use the currentPerson subject for this